### ISSUE TITLE
Demographics- for patient identifiers per page

### ISSUE THREAD

**Issue by abuckley3385** [department-of-veterans-affairs/va.gov-team#13153]
# Product Documentation
[Questionnaires Product Outline ](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/questionnaire/product/product-outline.md)
- [Visit Intro Initiative (Proof of Concept)](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/questionnaire/product/initiatives/visit-intro.md)
[Feature - Demographics ](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/questionnaire/product/features/demographics.md)

## User Story/ies

* Use Case: To ensure the user is viewing and documenting their own health information or proxy's all pages need to show at least 3 patient identifiers. 

* Description - Present up to 3 patient identifiers across all pages of the questionnaire. 
## Tasks
- [ ] See Demographics feature outline for fields and valuation
- [ ] Create detailed mockups and callouts of Demographics page [link] 
- [ ] Review with Accessibility and feedback incorporated or documented for team review 

## Acceptance Criteria
- [ ] UI review meeting with product for feature capabilities 
- [ ] UI review meeting and callouts with engineer for layout and callouts
- [ ] published Link and updated product feature outline 
- [ ] Access-ability review complete

**Comment by abuckley3385** [comment]
Thread in platform - https://dsva.slack.com/archives/C01CJV0L9PS/p1605634685052900

**Comment by kristenmcconnell** [comment]
@mlighdc  Regarding our additional privacy questions...

1. I logged into staging and found the following two identical examples of a checkbox to read and accept the privacy policy before submitting a form. Do we need to incorporate a similar checkbox? 

![Screen Shot 2020-12-04 at 1.22.44 PM.png](https://images.zenhubusercontent.com/5f44055c65a5c54f7cfe5928/43ad0742-ed49-48db-8e6d-271c5c30baa5)
![Screen Shot 2020-12-04 at 1.23.38 PM.png](https://images.zenhubusercontent.com/5f44055c65a5c54f7cfe5928/374514e3-64df-4e04-85b2-b7916ef0c1bf)

2. Do we need to show additional personal identifiers within the UI when displaying health data? If so, this might involve a change to the header of va.gov, such as the addition of a photo or another piece of information (last name, gender, or DOB). We could also put this additional identifier within the content section of health care pages. We just need a decision on the number of identifiers we should have on the UI. This becomes a bigger deal when we add proxy access onto va.gov. Other examples of personal identifiers in other health UIs:
- Blue button has name and photo in the header.
![Image from iOS.png](https://images.zenhubusercontent.com/5f44055c65a5c54f7cfe5928/0c03055c-1f1e-4dd7-898f-da40084be067)
- MyUPMC (a healthcare system's patient portal in Pittsburgh) has the patient's initial or photo in the header, so only one identifier.
![Screen Shot 2020-10-28 at 4.12.31 PM.png](https://images.zenhubusercontent.com/5f44055c65a5c54f7cfe5928/9dff99e0-281d-4efe-924b-ff3dd5c1313f)
- myHealtheVet's site header has the Veteran's first name. Most of the medical record content areas follow this pattern...main lists (such as all test results) only have the first name in the header, but detail pages (results from one particular test) have the Veteran's full name listed in the content area as well as the first name in the header. Then, the medications list follows its own pattern. It has the Veteran's full name and date of birth in the main content area.

**Comment by abuckley3385** [comment]
@mlighdc did you have a chance to followup on the demographics identifiers with the privacy/security team in Va.gov site vs. the pdf. ?

**Comment by mlighdc** [comment]
Yes, I was able to follow-up @abuckley3385 . The footer is fine but we need to work on the "sharing" statement. It is on my list to follow-up this week.

**Comment by kristenmcconnell** [comment]
@Ming I know content had the "sharing information" on their list of things to look into as well. I know Danielle was part of the COVID work, so she probably hasn't looked into it. Any feedback on the additional questions that I posed in the December 4 comment above?

**Comment by mlighdc** [comment]
@kristenmcconnell thanks for the reminder. I will need to follow up on #1. For #2, it looks like there are potentially two device type to account... on mobile and on laptop/desktop. I do not remember seeing the picture when using Blue Button on laptop/desktop. Let me know if you are also seeing on laptop / desktop.

**Comment by kristenmcconnell** [comment]
@ming It looks like the Blue Button pages (where a Veteran can select what type of exact report they want and download the reports) on desktop have the Veteran's full name and the branch of military that they served in within the UI.

**Comment by kristenmcconnell** [comment]
@mlighdc @leahkeeler Following up on this ticket that is still in the blocked column. Ming - Did we answer question #1 from the comment on Dec. 4 (shown below)? Besides that, I think we answered everything in this ticket, so it could be closed.

1. I logged into staging and found the following two identical examples of a checkbox to read and accept the privacy policy before submitting a form. Do we need to incorporate a similar checkbox? 

![Screen Shot 2020-12-04 at 1.22.44 PM.png](https://images.zenhubusercontent.com/5f44055c65a5c54f7cfe5928/43ad0742-ed49-48db-8e6d-271c5c30baa5)
![Screen Shot 2020-12-04 at 1.23.38 PM.png](https://images.zenhubusercontent.com/5f44055c65a5c54f7cfe5928/374514e3-64df-4e04-85b2-b7916ef0c1bf)
